ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/mcs-client.md

```diff
@@ -53,9 +53,9 @@ interface Management1
 
 MCS client is enabled
 
-| Secondary CVX cluster | Server Hosts |
-| --------------------- | ------------ |
-| default | 10.90.224.188, 10.90.224.189, leaf2.atd.lab |
+| Secondary CVX cluster | Server Hosts | Enabled |
+| --------------------- | ------------ | -------- |
+| default | 10.90.224.188, 10.90.224.189, leaf2.atd.lab | False |
```

Comment:
The enabled column is not correct compared to the config. `False` here but `no shutdown` in the config.